### PR TITLE
Template added for studio instance needed

### PR DESCRIPTION
**Pull request by tasawernawaz**
[ECOM-6097](https://openedx.atlassian.net/browse/ECOM-6097)

### CODE REVIEW

**Comment by tasawernawaz** [comment]
@awais786 @waheedahmed Please review.

File: course_discovery/templates/publisher/email/studio_instance_needed.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  <p>
+    {% trans "Dear," %} {{ edx_pc_name }}
+  </p>
+  <p>
+    {% trans "Please create a Studio instance for the following course." %}
+  </p>
+
+  <ul>
+    <li>{{ course_name }}</li>
```
Comment (waheedahmed) [inline course_discovery/templates/publisher/email/studio_instance_needed.html]: Where is the link to dashboard?
```<course name> (provide a link to the dashboard)```

File: course_discovery/templates/publisher/email/studio_instance_needed [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  <p>
+    {% trans "Dear," %} {{ edx_pc_name }}
+  </p>
+  <p>
+    {% trans "Please create a Studio instance for the following course." %}
+  </p>
+
+  <ul>
+    <li>{{ course_name }}</li>
```
Comment (waheedahmed) [inline course_discovery/templates/publisher/email/studio_instance_needed]: Where is the link to dashboard?
```<course name> (provide a link to the dashboard)```

File: course_discovery/templates/publisher/email/studio_instance_needed [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{% load i18n %}
+
+  <p>
```
Comment (waheedahmed) [inline course_discovery/templates/publisher/email/studio_instance_needed]: Fix indentation.

**Review by waheedahmed — CHANGES_REQUESTED** [review]
This looks good, just minor changes.

File: course_discovery/templates/publisher/email/studio_instance_needed [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    {% trans "Please create a Studio instance for the following course." %}
+  </p>
+
+  <ul>
+    <li>{{ course_name }}</li>
+    <li>{{ insitution_name }}</li>
+  </ul>
+
+  <p>
+    {% trans "Thanks," %} {{ course_team_admin_name }}
```
Comment (waheedahmed) [inline course_discovery/templates/publisher/email/studio_instance_needed]: ```course_team_admin_name``` on next line.

File: course_discovery/templates/publisher/email/studio_instance_needed.html [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    <li>{{ insitution_name }}</li>
+  </ul>
+
+  <p>
+    {% trans "Thanks," %} {{ course_team_admin_name }}
```
Comment (waheedahmed) [inline course_discovery/templates/publisher/email/studio_instance_needed.html]: Same here, ```course_team_admin_name``` on next line.

**Comment by tasawernawaz** [comment]
@waheedahmed Made suggested changes.

File: course_discovery/templates/publisher/email/studio_instance_needed [hunk 6]
```diff
@@ -0,0 +1,18 @@
+{% load i18n %}
+
+<p>
+{% trans "Dear," %} {{ edx_pc_name }}
```
Comment (waheedahmed) [inline course_discovery/templates/publisher/email/studio_instance_needed]: Comma after PC name.

File: course_discovery/templates/publisher/email/studio_instance_needed.html [hunk 7]
```diff
@@ -0,0 +1,23 @@
+{% extends "publisher/email/email_base.html" %}{% load i18n %}
+{% block body %}
+  <!-- Message Body -->
+
+  <p>
+    {% trans "Dear," %} {{ edx_pc_name }}
```
Comment (waheedahmed) [inline course_discovery/templates/publisher/email/studio_instance_needed.html]: Comma after PC name.

**Review by waheedahmed — APPROVED** [review]

**Comment by tasawernawaz** [comment]
@bderusha Please review this.

File: course_discovery/templates/publisher/email/studio_instance_needed [hunk 8]
```diff
@@ -0,0 +1,18 @@
+{% load i18n %}
```
Comment (awais786) [inline course_discovery/templates/publisher/email/studio_instance_needed]: Whats the type of this file ?

Comment (tasawernawaz) [reply course_discovery/templates/publisher/email/studio_instance_needed]: .txt

**Review by awais786 — APPROVED** [review]

**Pull request merged**